include/rocksdb/utilities/transaction.h

```diff
@@ -287,6 +293,9 @@ class Transaction {
   // functions in WriteBatch, but will also do conflict checking on the
   // keys being written.
   //
+  // assume_tracked=false expects the key be already tracked. If valid then it
```

Comment:
I think you meant assume_tracked=true here.

Reply:
oops. will fix it.